datafusion/core/tests/physical_optimizer/limit_pushdown.rs

```diff
@@ -52,18 +52,9 @@ fn projection_exec(
 ) -> Result<Arc<dyn ExecutionPlan>> {
     Ok(Arc::new(ProjectionExec::try_new(
         vec![
-            ProjectionExpr {
-                expr: col("c1", schema.as_ref()).unwrap(),
-                alias: "c1".to_string(),
-            },
-            ProjectionExpr {
-                expr: col("c2", schema.as_ref()).unwrap(),
-                alias: "c2".to_string(),
-            },
-            ProjectionExpr {
-                expr: col("c3", schema.as_ref()).unwrap(),
-                alias: "c3".to_string(),
-            },
+            (col("c1", schema.as_ref()).unwrap(), "c1".to_string()),
```

Comment:
This reverts a change made to the previous example to show it could still be used

Reply:
Added